public-ui/app.yaml

```diff
@@ -0,0 +1,19 @@
+runtime: python27
+api_version: 1
+threadsafe: true
+default_expiration: "2m"
+
```

Comment:
Per slack discussion, add

`service: public-ui`

This shows up in the subdomain prefix for app engine. The normal "ui" directory doesn't need this since we wanted to deploy it as the default app engine version (no prefix on the app engine subdomain)

Reply:
Added.